packages/components/modules/messages/GroupDetails/ProfileCard/constants.ts

```diff
@@ -0,0 +1,4 @@
+export enum ChatRoomParticipantRoles {
```

Comment:
would be cool to not use anums and prefers a constant + type instead
```suggestion
import { ValueOf } from '@baseapp-frontend/utils'

export const CHAT_ROOM_PARTICIPANT_ROLES = {
  admin: 'ADMIN',
  member: 'MEMBER',
} as const

export type ChatRoomParticipantRoles = ValueOf<typeof CHAT_ROOM_PARTICIPANT_ROLES>
```

Reply:
enums can have some tradeoffs and little [bugs](https://github.com/microsoft/TypeScript/issues?q=is%3Aissue+is%3Aopen+enum+label%3Abug), constants + types offers more stability and doesnt need to be transpilled to code

Reply:
Good suggestion, thanks!